app/Jobs/Curations/SyncPhenotypes.php

```diff
@@ -4,27 +4,55 @@
 
 use App\Phenotype;
 use App\Curation;
+use App\Events\Curation\CurationPhenotypesUpdated;
 use Illuminate\Bus\Queueable;
 use Illuminate\Contracts\Queue\ShouldQueue;
 use Illuminate\Foundation\Bus\Dispatchable;
 use Illuminate\Queue\InteractsWithQueue;
 use Illuminate\Queue\SerializesModels;
+use Illuminate\Validation\ValidationException;
 
 class SyncPhenotypes implements ShouldQueue
 {
     use Dispatchable, InteractsWithQueue, Queueable, SerializesModels;
 
-    public function __construct(private Curation $curation, private $phenotypeIds)
+    private array $errors = [];
+
+    public function __construct(private Curation $curation, private array $phenotypeIds)
     {
     }
 
     public function handle()
     {
-        $phenotypes = Phenotype::whereIn('id', $this->phenotypeIds ?? []);
-        if (!$phenotypes && $phenotypes->count() > 0) {
+        if (!$this->phenotypeIdsAreValid()) {
+            throw ValidationException::withMessages($this->errors);
+        }
+
+        $previousIds = $this->curation->phenotypes->pluck('id');
+
+        if ($previousIds->toArray() == $this->phenotypeIds) {
```

Comment:
technially `sync` below does nothing when the relations being synced haven't changed.  Now that we're firing the event after the sync, it makes sense to check for equality before calling sync, to ensure we don't capture redundant events.